python/taichi/lang/__init__.py

```diff
@@ -103,9 +103,14 @@ def add(self, key, cast=None):
         value = os.environ.get(name, '')
         if len(value):
             self[key] = cast(value)
+            if key in self.kwargs:
+                core.warn(
```

Comment:
I think ti.info is good enough?

Reply:
I still think `warn` is necessary, since if a user ignores this message he may run into unexpected behaviors. WDYT?